src/client/cockpit-beiboot

```diff
@@ -174,7 +174,8 @@ class AuthorizeResponder(ferny.InteractionResponder):
 
         if command == 'cockpit.report-exists':
             file_status, = args
-            self.router.packages = Packages(loader=ProxyPackagesLoader(file_status))
+            # FIXME: evil duck typing here -- this is a half-way Bridge
+            self.router.packages = Packages(loader=ProxyPackagesLoader(file_status))  # type: ignore[attr-defined]
```

Comment:
This is another case of

```python
        packages: Packages = self.router.packages  # type: ignore[attr-defined]  # yes, this is evil
```

from `src/cockpit/channels/packages.py`.

We really ought to figure out a better way...

Reply:
Yes, I mentioned that in the commit message. I was hoping you had a better idea..